runtime/src/main/kotlin/kotlin/native/concurrent/Worker.kt

```diff
@@ -103,6 +103,18 @@ public inline class Worker @PublishedApi internal constructor(val id: Int) {
         executeAfterInternal(id, operation, afterMicroseconds)
     }
 
+
+    /**
+     * Process pending job(s) on the queue of this worker, returns `true` is something was processed
```

Comment:
> returns `true` is something

typo.

Reply:
> Process pending job(s)

Can it really process multiple jobs?

Reply:
Yes, if inside a job there's call to processQueue.

Reply:
Then this would be the effect of the job, not that of `processQueue` directly.
This side effect is worth mentioning explicitly, leaving here singular "job".

Reply:
Do not think so.